test/explain/algorithm/test_captum_explainer_heterogeneous.py

```diff
@@ -0,0 +1,254 @@
+from typing import Literal, Optional
+
+import pytest
+import torch
+
+from torch_geometric.explain import Explainer, HeteroExplanation
+from torch_geometric.explain.algorithm import CaptumExplainer
+from torch_geometric.explain.config import (
+    MaskType,
+    ModelConfig,
+    ModelMode,
+    ModelReturnType,
+    ModelTaskLevel,
+)
+from torch_geometric.nn import HANConv, HGTConv, global_add_pool
+from torch_geometric.testing import withPackage
+from torch_geometric.transforms import ToSparseTensor
+
+methods = [
```

Comment:
I mostly kept them to be consistent with the other Captum test files and because in theory one could change the test to use all the supported methods - we just demurred from doing so because it makes the tests overall take much longer. I'm happy to remove them if you really want to do so.